statusroutes_test.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStatusRoutes(testCase *testing.T) {
+	testRouter := mux.NewRouter()
+	serviceName := "%CUSTOM_PLUGIN_SERVICE_NAME%"
+	serviceVersion := "0.0.0"
+	StatusRoutes(testRouter, serviceName, serviceVersion)
+
+	testCase.Run("/-/healthz - ok", func(assert *testing.T) {
+		expectedResponse := fmt.Sprintf("{\"status\":\"OK\",\"name\":\"%s\",\"version\":\"%s\"}", serviceName, serviceVersion)
+		responseRecorder := httptest.NewRecorder()
+		request, error := http.NewRequest(http.MethodGet, "/-/healthz", nil)
+		if error != nil {
+			assert.Fatal("Error creating the /-/healthz request")
+		}
```

Comment:
This could be replaced with:
```suggestion
		require.NoError(assert, err, "Error creating the /-/healthz request")
```
I don't know if assert is correct, the convention is to use t. It does not have any assertion methods

Reply:
The same for other tests

Reply:
`assert` renamed to `t` and `if` statement changed with `require.noError`